src/test/java/com/example/external/data/CarRepositoryIntegrationTests.java

```diff
@@ -30,7 +30,9 @@
  * @author Mark Paluch
  */
 @RunWith(SpringJUnit4ClassRunner.class)
-@SpringBootTest
+// no need to load the full context
+//@SpringBootTest
+@DataJpaTest
```

Comment:
Thanks a lot. Can we remove the commented code?

Reply:
Yes. Thanks.